build.gradle

```diff
@@ -168,7 +185,8 @@ dependencies {
     testCompile 'org.reflections:reflections:0.9.11'
     testCompile 'org.xmlunit:xmlunit-core:2.5.1'
     testCompile 'org.xmlunit:xmlunit-matchers:2.5.1'
-    testCompile 'com.tngtech.archunit:archunit-junit:0.5.0'
+    // TODO: has a split package between archunit and archunit-junit
+//    testCompile 'com.tngtech.archunit:archunit-junit:0.5.0'
```

Comment:
I just discovered that they  converted that to  java9 compatibility in a later snapshot:  https://github.com/TNG/ArchUnit/issues/47